maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -148,10 +114,10 @@ public fun rememberMarkerState(
 @Composable
 @GoogleMapComposable
 public fun Marker(
+    position: LatLng,
     state: MarkerState = rememberMarkerState(),
     alpha: Float = 1.0f,
     anchor: Offset = Offset(0.5f, 1.0f),
-    draggable: Boolean = false,
```

Comment:
What's the reason for removing this param? I would expect this to still be here.

Reply:
I was thinking the same thing - we still need it to indicate if the user _can_ drag a marker

Reply:
Thanks, having `DragState` available via an event handler will be useful.

In #149 I laid out in detail why I find it both necessary and feasible to also eliminate `position` from `MarkerState`. In summary, it is true that the Maps SDK owns the marker's position, but closely aligning the Compose API design with that ownership model is barely usable except in toy apps. I linked several issues caused by this design that came up since I created #149, and I am certain that this pattern will repeat until the design changes.

I agree that it is not ideal and means taking a small risk to turn the ownership model on its head, but the current approach is just too awful to use, every time I look at my workarounds there I get a stomach ache. A design change here could be confined to an experimental playground somehow.

I'd likely try to run my own fork if it cannot be resolved within the main project.

`CameraPositionState` is very different because there is only one camera. Hoisting it into a State object does not have a great impact on usability.

Reply:
I do think that `onDragEvent` should receive the current Marker position as a parameter, as I suggested in my original PR. Otherwise the `onDragEvent` lambda would need to rely on an implicit promise that MarkerState is timely updated to the position tied to the corresponding drag event. This is obscure on the API consumer's side, and would seem error-prone on the API implementation side.